.github/labeler.yml

```diff
@@ -0,0 +1,48 @@
+batch:
+  - '(batch)'
+bug:
+  - '(bug)'
+config.writer:
+  - '(config.writer)'
+documentation:
+  - '(documentation)'
+duplicate:
```

Comment:
I don't know if this is needed? I think a duplicate label will get added after the fact since folks wouldn't knowingly create a duplicate. I suspect this might catch other descriptions like "So and so method returns duplicate results" or similar.